src/main/features/claim/form/models/interestStartDate.ts

```diff
@@ -7,6 +7,10 @@ import { IsValidYearFormat } from 'app/forms/validation/validators/isValidYearFo
 import { ValidationErrors as CommonValidationErrors } from 'app/forms/validation/validationErrors'
 import { CompletableTask } from 'models/task'
 
+export class ValidationErrors {
+  static readonly REASON_REQUIRED: string = 'You need to explain why you\'re claiming from a particular date'
```

Comment:
plz use apostrophe  i.e. ` you’re`  rather than `you\'re`

Reply:
Good catch, changing it now.